server/src/main/java/org/solarbank/server/ErrorMessage.java

```diff
@@ -7,7 +7,8 @@ public enum ErrorMessage {
     INTERNAL_SERVER_ERROR("An unexpected error occurred. Please try again later."),
     NO_FIELD_ERRORS("Field errors not found"),
     NO_ERROR_MESSAGE("no error captured"),
-    NOT_FOUND("Page not found: ");
+    NOT_FOUND("Page not found: "),
+    TEST_FAIL("failed to perform test api requests as expected");
```

Comment:
We shouldn't have this in the production code as it's a test specific message. Just make it a variable in the test file.